library/erlang

```diff
@@ -3,16 +3,16 @@
 Maintainers: Mr C0B <denc716@gmail.com> (@c0b)
 GitRepo: https://github.com/c0b/docker-erlang-otp.git
 
-Tags: 20.0, latest
-GitCommit: 8128be859f47c39473be9fa2c2ae33af28f74279
+Tags: 20.0.1, latest
```

Comment:
@getong I've just realized that the last PR to docker-library wasn't run from https://github.com/c0b/docker-erlang-otp/blob/25224b9/generate-stackbrew-library.sh ? so that the tag 20.0 is missing; if run from the script, it will generate something like Tags: 20.0.1 20.0 20 latest

so that if pull erlang:20 image at the moment, it is still pointing the 20.0 image not the 20.0.1 image; this is a problem.

so please run from /generate-stackbrew-library.sh if you do the PR next time.